CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ------------------
+### Added
+* Generate CHANGELOG entry in docs
```

Comment:
Can we update this to indicate we added a Changelog template for the terra-dev-site docs setup